modules/ROOT/partials/configuration/templates.adoc

```diff
@@ -7,6 +7,8 @@ If this option is a string then it will be requested as a URL that should produc
 
 Each item in the list can either be inline using a `+content+` property or a whole file using the `+url+` property.
 
+Type : `+String+` or `+Array+` or `+Function+`
```

Comment:
```suggestion
Type : `+String+`, `+Array+`, or `+Function+`
```

Reply:
Also do we need examples for these different types? It looks like all we're missing here is a function which is probably fine to skip.

Reply:
Have added snippet for missing function example